CHANGELOG.md

```diff
@@ -90,10 +90,11 @@
 ### Fixed 
 * Optional dependencies to support M1 installation
 
-## 1.2.4 5/03/2022
+## 2.0.0 5/20/2022
 
 ### Added
 
 * New feature in FileProcessing class to read and return file as json
 * New feature in Highlighter class to redact and replace highlights with spoofed data
-* Upgrades client to 5.1.1
\ No newline at end of file
+* New Download class to support downloading resources from an Indico Cluster
+* Upgrades client to 5.1.3
```

Comment:
And explicitly call out that SDK calls have been upgraded for Indico 5.X compatibility 
Probably should add ### Removed Finder